pkg/logs/message/origin.go

```diff
@@ -34,8 +34,8 @@ func (o *Origin) Tags() []string {
 	if o.LogSource.Config.SourceCategory != "" {
 		tags = append(tags, "sourcecategory:"+o.LogSource.Config.SourceCategory)
 	}
-	if o.LogSource.Config.Tags != "" {
-		tags = append(tags, strings.Split(o.LogSource.Config.Tags, ",")...)
+	if len(o.LogSource.Config.Tags) != 0 {
+		tags = append(tags, o.LogSource.Config.Tags...)
```

Comment:
go noob question: do we still need the `if` statement?

Reply:
We don't :-)